### ISSUE TITLE
Accessibility signal for finished/failed tests

### ISSUE THREAD

**Issue by jacekkopecky** [microsoft/vscode#254603]
Please add accessibility signals for when tests succeed or fail.

When developing tests, I want to be able to run the tests frequently and be notified when they succeed or fail. Front-end tests with typescript on a weak dev container take long seconds that I don't want to spend watching the subtle indicator that the test is running. My poor eyesight makes watching the indicator a bit difficult.

I have currently worked around this by using the multiCommand extension and doing a terminal beep after the test command finishes, but this works with an unnecessary delay and doesn't support continuous testing.

I believe there should be an accessibility signal for tests finishing (different for success and fail), which should also play when continuous testing reruns a test.

**Comment by meganrogge** [comment]
Tests are contributed by extensions, so this is not something we'd implement in core. Pls create issues for your test extensions related to this.

**Comment by jacekkopecky** [comment]
Thanks @meganrogge , but I didn't find a way for extensions to contribute accessibility signals, have I missed it somewhere?

**Comment by meganrogge** [comment]
Thanks for asking. You have not missed it - there's currently no API for this as extensions are also able to play sounds with relative ease like we do 

https://github.com/microsoft/vscode/blob/8a5452b013d0152adc11c71b25e9d74f21863cca/src/vs/platform/accessibilitySignal/browser/accessibilitySignalService.ts#L281-L288

**Comment by jacekkopecky** [comment]
Thanks @meganrogge , I didn't know, that's nice! 

Still, I see one more straw to grasp ( :smile: ) – testing seems to be core in VSCode, at least the testing panel seems to be common between Python and Vitest in my recent experience. Is it because both extensions implement it the same way?

If there is core support for testing, I think common accessibility signals would also make sense, so they are easily common between different languages.

**Comment by meganrogge** [comment]
@connor4312 is there some place in core that we handle success / failure of tests? If so, then it would be possible and a good thing to do generically
